specs2/specs2.bzl

```diff
@@ -4,36 +4,36 @@ def specs2_repositories():
 
   native.maven_jar(
       name = "io_bazel_rules_scala_org_specs2_specs2_core",
-      artifact = "org.specs2:specs2-core_2.11:" + specs2_version(),
-      sha1 = "495bed00c73483f4f5f43945fde63c615d03e637",
+      artifact = "org.specs2:specs2-core_2.12:" + specs2_version(),
```

Comment:
if we used `scala_mvn_artifact` we wouldn't have this change here.

Reply:
I went wholesale on this in #303